kolibri/plugins/user_profile/viewsets.py

```diff
@@ -17,14 +16,12 @@ class UserIndividualViewset(APIView):
     def get(self, request, format=None):
         if request.user.is_anonymous:
             self.permission_denied(request)
-
         subset_of_users_device = get_device_setting(
             "subset_of_users_device", default=False
         )
-        users_device = FacilityUser.objects.count()
-
+        user_facility = self.request.user.facility
         return Response(
-            {"individual": users_device == 1, "lod": subset_of_users_device}
+            {"individual": user_facility.on_my_own_setup, "lod": subset_of_users_device}
```

Comment:
Would it make sense to use "on my own setup" naming instead of "individual"? So that we don't have more terms in a product where there are already lots of them.

Reply:
yes, as  you prefer. I just didn't want to change too many things in the current code, this was the cleanest approach but you're right, it drags the previous logic in the naming. In case you prefer it, I'll rename the composable and refactor it with en `onMyOwnSetup` variable instead of individual

Reply:
I'm not sure what's my preference since these are my first encounters with various kinds of setups (and probably that's why using more terms makes things a bit confusing for me). I'd go with whatever we officially/most commonly use? Also, in regards to this PR, it's nothing blocking.

Reply:
Yeah, I think just up date to `on_my_own_setup` for the response in a follow up PR.